packages/design-system/src/components/Alert/Alert.styled.ts

```diff
@@ -0,0 +1,66 @@
+import {
+  Alert as MuiAlert,
+  AlertTitle as MuiAlertTitle,
+  IconButton as MuiIconButton,
+  styled,
+} from "@mui/material";
+import { AlertProps } from "./Alert.types";
```

Comment:
```suggestion
import types { AlertProps } from "./Alert.types";
```

type을 가져오는 경우 명시적을 type을 붙히는 것으로 클래스와의 혼동을 피할 수 있고, 컴파일시 이점도 있기 때문에 사용해보시는 건 어떨까요